app/assets/javascripts/uswds.js

```diff
@@ -0,0 +1,14 @@
+var button = document.getElementsByClassName("usa-banner__button")[0];
+var banner_content = document.getElementsByClassName("usa-banner__content")[0];
+
+button.addEventListener("click", function(){
+  var is_hidden = banner_content.getAttribute("hidden");
+  if(is_hidden){
+    banner_content.removeAttribute("hidden");
+    this.setAttribute("aria-expanded", "true");
+  }
+  else{
+    banner_content.setAttribute("hidden", "true");
+    this.setAttribute("aria-expanded", "false");
+  }
+});
```

Comment:
Totally up to your preferences, but this made me think of `toggleAttribute`: 

```suggestion
button.addEventListener("click", function(){
  banner_content.toggleAttribute("hidden");
  this.setAttribute("aria-expanded",
    this.getAttribute("aria-expanded") === "true"
      ? "false"
      : "true"
  );
});
```

Reply:
Thanks @danswick - this is a good suggestion. I will update my code accordingly.

Reply:
Hi @danswick I have taken care of your suggestions.